changelog.d/653.change.rst

```diff
@@ -0,0 +1,5 @@
+``attr.s()`` now has a *transform_hook* that is called for each
```

Comment:
We use [semantic newlines](http://rhodesmill.org/brandon/2012/one-sentence-per-line/). :)